source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -31,7 +31,14 @@ For example:
 
 All the parameters in `billing_address` are optional.
 
-If you include the `country` parameter, it must be an <a href="https://www.iso.org/iso-3166-country-codes.html" target="blank">ISO 3166 code</a>. If the parameter is invalid or missing, the __Country or territory__ field on the payment page will default to 'Great Britain'.
+The values must be no longer than:
+
+- 254 characters for `email`
+- 255 characters for `cardholder_name`
+- 255 characters each for `line1`, `line2` and `city` in `billing_address`
+- 25 characters for `postcode` in `billing_address`
+
+If you include the `country` parameter, it must be an <a href="https://www.iso.org/obp/ui/#search/code/" target="blank">ISO 3166-1 Alpha-2 code</a>. If the parameter is invalid or missing, the __Country or territory__ field on the payment page will default to 'Great Britain'.
```

Comment:
I think you want `target="blank"` (open in a browsing context called `blank`) to be `target="_blank"` (open in a new browsing context). Also, it seems it’s always written “alpha-2” with a lower-case “a”. Finally, the default is actually the United Kingdom rather than Great Britain.

Reply:
Ah thanks! We used `target="blank"` elsewhere because it worked - but looks like that will only work if you don't have an existing tab called 'blank'.
I thought it was 'Alpha-2' because ISO had written it that way on https://www.iso.org/obp/ui/#search/code/, but in retrospect that seems to be capitalisation just for their table heading? Will correct!

Reply:
I've fixed those now in https://github.com/alphagov/pay-tech-docs/pull/300/commits/0ba6547dbd0d68f8fd659f736cd844a3118c1726.

Reply:
I hadn’t seen that table. I think you’re right they used capital letters at the beginning because they were treating it like the first word is a title. On https://www.iso.org/iso-3166-country-codes.html in running text they use “alpha-2” and “alpha-3” etc.